README.md

```diff
@@ -78,25 +82,83 @@ Run the following steps on your laptop or local workstation:
         cd dockerflow
         mvn package -DskipTests
 
-3. Set up the DOCKERFLOW_HOME environment.
+3.  Set up the DOCKERFLOW_HOME environment.
 
         export DOCKERFLOW_HOME="$(pwd)"
         export PATH="${PATH}":"${DOCKERFLOW_HOME}/bin"
         chmod +x bin/*
 
-4.  Run a sample workflow:
+4.  Set up your local project environmental variables.
```

Comment:
Looks to me like the friction here is that users don't know their project name and might not have a bucket yet. Also, by setting variables, users can copy-paste the examples. Is that right? 

Let's ask Matt & Nicole about their feeling on setting new env variables. I haven't seen us doing that much in our docs. It's true that requiring users to edit the command-lines is kind of annoying :)